src/components/performances/PerformancesSection.jsx

```diff
@@ -20,25 +23,21 @@ class PerformancesSection extends React.Component {
   }
 
   componentDidMount() {
-    const { match: { params: { eventId }}} = this.props;
-    const collectionName = `events/${eventId}/performances`;
-
-    this.subscribe = db.collection(collectionName).onSnapshot((querySnapshot) => {
-      let performances = [];
-      querySnapshot.forEach((doc) => {
-        const performance = {
-          id: doc.id,
-          ...doc.data()
-        };
-        performances.push(performance);
-      });
-      performances = performances.sort((a, b) => Number(a.danceEntry) - Number(b.danceEntry));
-      this.setState({ filteredPerformances: performances, loading: false, performances });
-    });
-  }
-
-  componentWillUnmount() {
-    this.subscribe();
+	const { match: { params: { eventId }}} = this.props;
+	let performances = [];
+	
+	getPerformances(eventId)
+	.then(response => {
+		Object.values(response.data).forEach(performance => {
+			performances.push(humps.camelizeKeys(performance));
+		});
+		performances = performances.sort((a,b) => Number(a.danceEntry) - Number(b.danceEntry));
```

Comment:
is there a way to sort this by just casting each `danceEntry` to a `Number`?? i.e. ```performances = performances.sort((performance) => Number(performance.danceEntry));```
If not, then leave it how it is